tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -176,7 +177,7 @@ public static void Initialize()
                 }
                 catch (Exception e)
                 {
-                    Log.Error(e, e.Message);
+                    Log.Error(e, "Failed to initialize Remote Configuration Management.");
```

Comment:
Could be good to keep the exception message too, to help understand what happened, along with where it happened?

Reply:
The message is always included when you pass an `Exception` to a `Log` method, so `e.Message` is always unnecessary duplication 🙂

Reply:
Resolving as I believe Andrew is correct.